packages/react/src/drafts/TabPanels/TabPanels.tsx

```diff
@@ -0,0 +1,161 @@
+import type {ForwardRefComponent as PolymorphicForwardRefComponent} from '../../utils/polymorphic'
+import clsx from 'clsx'
+import React from 'react'
+import styled from 'styled-components'
+import {get} from '../../constants'
+import {TabContainerElement} from '@github/tab-container-element'
+import {createComponent} from '../../utils/custom-element'
+import sx, {type SxProp} from '../../sx'
+import type {ComponentProps} from '../../utils/types'
+import getGlobalFocusStyles from '../../internal/utils/getGlobalFocusStyles'
+
+const TAB_CLASS = 'TabPanel-tab'
+
+const TabContainer = createComponent(TabContainerElement, 'tab-container')
+const StyledTabContainer = styled(TabContainer)`
```

Comment:
@keithamus let me know if there's a better way to do this. Initially I tried wrapping the previous line but it gave me an error:

> Type instantiation is excessively deep and possibly infinite

Moving the call to a separate line seems to confuse the TypeScript typing stuff less 🤔